Source/WebKit/Shared/Cocoa/CoreIPCDictionary.mm

```diff
@@ -50,13 +51,76 @@
     }
 }
 
-RetainPtr<id> CoreIPCDictionary::toID() const
+bool CoreIPCDictionary::keyHasValueOfType(const String& key, IPC::NSType type) const
+{
+    createNSDictionaryIfNeeded();
+    id object = [m_nsDictionary.get() objectForKey:(NSString *)key];
```

Comment:
(NSString *) is technically not necessary because of String::operator NSString *, but some people think it's nice.